src/transformers/generation/logits_process.py

```diff
@@ -322,11 +328,18 @@ def __call__(self, input_ids: torch.LongTensor, scores: torch.FloatTensor) -> to
 
 class EncoderRepetitionPenaltyLogitsProcessor(LogitsProcessor):
     r"""
-    [`LogitsProcessor`] enforcing an exponential penalty on tokens that are not in the original input.
+    [`LogitsProcessor`] enforcing an exponential penalty on tokens that are not in the original input. This technique
+    avoids hallucination by boosting the probabilities of tokens found within the original input.
+
+    While the intended usage is to penalize and thus reduce hallucination, this technique can also be used to reward and
+    thus encourage creativity in a similar manner. To penalize and reduce repetition, use `hallucination_penalty` values
```

Comment:
```suggestion
    This technique can also be used to reward and thus encourage creativity in a similar manner. To penalize and reduce repetition, use `hallucination_penalty` values
```
(same comment as above)